google/cloud/bigtable/client_options.h

```diff
@@ -37,15 +37,26 @@ class ClientOptions {
   /**
    * Initialize the client options with the default credentials.
    *
-   * If the `BIGTABLE_EMULATOR_HOST` environment variable is set this
-   * constructor configures the object to connect to the host and port set in
-   * that environment variable. If the environment variable is not set, this
-   * constructor configures the object to connect to the production instance of
-   * Cloud Bigtable using the application default credentials.
+   * Configure the client to connect to the production instance of Cloud
```

Comment:
We use the term "production instance" to mean a specific type of Cloud Bigtable instance (as opposed to a "development instance") but they're the same from the API perspective. I presume here you mean "Cloud Bigtable service" vs. emulator, rather than "production".

Reply:
Yes, fixed.